services/provider/server/storageclaim.go

```diff
@@ -20,20 +20,20 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 )
 
-type storageClassClaimManager struct {
```

Comment:
file needs to be renamed to storagerequest.go

Reply:
Why is this even called `stoageclaim`/`storagerequest`? aren't we supposed to call this `storageclassrequest`?